test/cpp/json-parse.cpp

```diff
@@ -0,0 +1,38 @@
+/*********************************************************************
+ * NAN - Native Abstractions for Node.js
+ *
+ * Copyright (c) 2017 NAN contributors
+ *
+ * MIT License <https://github.com/nodejs/nan/blob/master/LICENSE.md>
+ ********************************************************************/
+
+#include <nan.h>
+
+NAN_METHOD(Parse) {
+  Nan::JSON NanJSON;
+
+  Nan::MaybeLocal<v8::String> inp = Nan::To<v8::String>(info[0]);
+
+  if (inp.IsEmpty()) {
+    info.GetReturnValue().Set(Nan::Undefined());
```

Comment:
The default return value is `Undefined`, so there is no need to set it. Also, prefer the "fast primitive setters" where available, e.g. `ReturnValue::SetUndefined`. This applies in other places as well. Since the test suite has become a de facto example set and documentation, it is important to show best practices here.